src/core/github-api.js

```diff
@@ -0,0 +1,157 @@
+import { fetchAndCache } from "./utils";
+import { pub } from "./pubsubhub";
+export const name = "core/github-api";
+
+const MAX_GITHUB_REQUESTS = 60;
```

Comment:
We probably don't need this anymore, to be honest... we should be using the "requests left" information from github directly.